src/client/index.ts

```diff
@@ -186,17 +183,22 @@ export type CustomData = {[key: string]: any};
 export type Notification = Readonly<Required<Omit<NotificationOptions, 'buttons'>> & {readonly buttons: ReadonlyArray<Readonly<Required<ButtonOptions>>>}>;
 
 /**
- * Event fired for interactions with notification UI elements. It is important to note that applications will only
- * receive these events if they indicate to the service that they want to receive these events. See {@link Actions} for
- * a full example of how actions are defined, and how an application can listen to and handle them.
+ * Event fired when an action is raised for a notification UI elements. It is important to note that applications will
```

Comment:
>Event fired when an action is raised by notification UI elements.

Reply:
Actually, that sentence is just inaccurate. Fixed!